documentation/docs/src/user-guide/getting-started.md

```diff
@@ -15,9 +15,15 @@ The main binary `namada` has sub-commands for all of the other binaries:
 
 To explore the command-line interface, add `--help` argument at any sub-command level to find out any possible sub-commands and/or arguments.
 
+```admonish tip title="Adding binaries to path" collapsible=true
+The binaries should be added to `$PATH` from the `make install` command. However, if this for some reason did not work, a solution may be to copy the binaries from `namada/target/release` to `home/$USER/.local/bin/` for example:
```

Comment:
imo we should just link to the single source of truth installation guide, rather than including more installation instructions inline on this page. Same applies to the "Quickstart" guide which also mentions some installation instructions.

(These specific instructions here are OS dependent (this wouldn't work on macOS, I don't think) and require the user to know to `make build-release`/how to build release Rust binaries or similar first.)